src/actionTypes/index.js

```diff
@@ -0,0 +1,4 @@
+export const SET_IS_TOKEN_RECEIVED = 'SET_IS_TOKEN_RECEIVED';
+export const SET_TOKEN_EXPIRATION_TIME = 'SET_TOKEN_EXPIRATION_TIME';
+export const SET_IS_FETCHING_IN_PROGRESS = 'SET_IS_FETCHING_IN_PROGRESS';
+export const SET_IS_FETCHING_ERROR = 'SET_IS_FETCHING_ERROR';
```

Comment:
SET_FETCHING_ERROR ?

Reply:
Sorry, I did not catch the question:
1. If you ask to explain the logic of SET_IS_FETCHING_ERROR action - it sets isFetchingError flag to true or false value. This flag shows was any errors during data fetching (I'll set this flag if some of axios requests will be rejected)
2. If you mean that SET_FETCHING_ERROR  name is better than SET_IS_FETCHING_ERROR in this case - please write, I'll rename.

Reply:
I'd say there is not much useful info for error in a boolean